REST/README.md

```diff
@@ -0,0 +1,48 @@
+# REST-API
+
+## Overview
+
+This application is built using Go and MySQL and utilizes the Gin web framework. It allows you to perform CRUD operations on books.
+
+## Requirements
+
+- Go (Version 1.15 or higher)
+- MySQL
+
+## Setup
+
+### Environment Variables
+
+This application uses environment variables to handle configurations. To set these up:
+
+1. Create a .env file in the root directory of the application.
+2. Add your MySQL connection string as follows:
+
+```bash
+DB_HOST="localhost"
+DB_PORT="3307"
+DB_USER="user"
+DB_PASS="password"
+DB_NAME="db"
+
+```
+
+## Running the application
+
+After setting up the environment variables:
+
+1. Open a terminal and navigate to the root directory of the application.
+2. Run the following commands:
+
+```bash
+go mod download
+go run main.go
+```
+
+The application should now be running, and you should be connected to your MySQL database.
+
+### **Using Swagger to Test API**
+
+To test the API using Swagger:
+
+**Navigate to Swagger UI**: Open your web browser and go to **`http://localhost:8080/swagger/index.html`**.
```

Comment:
we can change this to be a link
[http://localhost:8080/swagger/index.html](http://localhost:8080/swagger/index.html)